bucket/wireguard.json

```
@@ -0,0 +1,38 @@
+{
+    "homepage": "https://www.wireguard.com",
+    "description": "Simple and fast VPN client.",
+    "version": "0.0.31",
+    "license": "GPL-2.0-or-later",
```

```suggestion
    "license": "MIT",
```

> The kernel components are released under the GPLv2, as is the Linux kernel itself. Other projects are licensed under MIT, BSD, Apache 2.0, or GPL, depending on context.

See https://github.com/WireGuard/wireguard-windows